### PR TITLE
Checkout Timetable

### PR DESCRIPTION
**Pull request by maxsebso**
## This PR

Added another timetable for the book check-out question. This PR is stuck until the backend updates the DTO for a report with library to include a `checkInTimetable` and `checkOutTimetable`.

## Screenshots

Screen Shot 2021-08-03 at 3 14 57 PM

### CODE REVIEW

File: src/components/report/ReportWithLibrary.tsx [hunk 1]
```diff
@@ -60,10 +71,14 @@ const ReportWithLibrary: React.FC<ReportWithLibraryProps> = ({
         <LibraryInfo editable={editable} />
         <MonitoringInfo
           editable={editable}
-          showTimeTable={showTimeTable}
-          setShowTimeTable={setShowTimeTable}
-          timeTable={timeTable}
-          setTimeTable={setTimeTable}
+          showCheckInTimeTable={showCheckInTimeTable}
```
Comment (rymaju) [inline src/components/report/ReportWithLibrary.tsx:74]: 😱 this sure is a lot of state, I wonder if we can abstract this out somehow?

**Review by rymaju — COMMENTED** [review]
Looks like solid work so far, theres some thunk tests that are failing or something though

File: src/components/report/ReportWithLibrary.tsx (hunk 1 above)
Comment (maxsebso) [reply src/components/report/ReportWithLibrary.tsx:74]: just we need to merge now and we can simplify later

**Review by rymaju — APPROVED** [review]
LGTM, needs to be refactored in the near future though

**Pull request merged**